webpack.config.coffee

```diff
@@ -0,0 +1,43 @@
+path = require("path")
+
+file_name = (mode) ->
+  if mode == "production"
+    "alephbet.min.js"
+  else
+    "alephbet.js"
+
+module.exports = (env, argv) ->
+  target: "web"
```

Comment:
You could leave this out since it is the default. Just wanted to mention it.

Reply:
Good to know. I usually prefer to keep things explicit unless they are really obvious/go against the convention...